package-lock.json

```diff
@@ -2256,6 +2256,7 @@
       "resolved": "https://registry.npmjs.org/prettier/-/prettier-3.2.5.tgz",
       "integrity": "sha512-3/GWa9aOC0YeD7LUfvOG2NiDyhOWRvt1k+rcKhOuYnMY24iiCphgneUfJDyFXd6rZCAnuLBv6UeAULtrhT/F4A==",
       "dev": true,
+      "license": "MIT",
```

Comment:
Revert the changes to this file.  Also, I recommend updating your release branch with the latest commits from `main` (it's currently 722 commits behind).